README.md

```diff
@@ -1,2 +1,6 @@
 # give_me_scales
-This CLI program gives you the notes of a scale. User inputs are the names of the scale and root note.
+This CLI program gives you the musical notes of a scale. User inputs are the names of the scale and root note.
+
+Quick-start:
+Clone the give_me_scales directory. In your terminal run:
+python main.py
```

Comment:
Please enclose code snippets and commands in three backticks, like this:

````
```
python main.py
```
````

which results in
```
python main.py
```


Inline code, for example `def f(x)`, is done with single backticks like this: `` `def f(x)` ``

Make headings with #, like this:

```
# Big heading
## Smaller heading
```
which renders as:

# Big heading
## Smaller heading


Github has a guide to Markdown: https://guides.github.com/features/mastering-markdown/